spec/locale.html

```diff
@@ -331,6 +347,87 @@ <h1>Intl.Locale.prototype.toString ( )</h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-Intl.Locale.prototype.getCalendars">
+      <h1>Intl.Locale.prototype.getCalendars ( )</h1>
+      <p>When the `getCalendars` method is called, the following steps are taken:</p>
+      <emu-alg>
+        1. Let _loc_ be the *this* value.
+        1. Perform ? RequireInternalSlot(_loc_, [[InitializedLocale]]).
+        1. Return CalendarsOfLocale(_loc_).
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-Intl.Locale.prototype.getCollations">
+      <h1>Intl.Locale.prototype.getCollations ( )</h1>
+      <p>When the `getCollations` method is called, the following steps are taken:</p>
+      <emu-alg>
+        1. Let _loc_ be the *this* value.
+        1. Perform ? RequireInternalSlot(_loc_, [[InitializedLocale]]).
+        1. Return CollationsOfLocale(_loc_).
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-Intl.Locale.prototype.getHourCycles">
+      <h1>Intl.Locale.prototype.getHourCycles ( )</h1>
+      <p>When the `getHourCycles` method is called, the following steps are taken:</p>
+      <emu-alg>
+        1. Let _loc_ be the *this* value.
+        1. Perform ? RequireInternalSlot(_loc_, [[InitializedLocale]]).
+        1. Return HourCyclesOfLocale(_loc_).
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-Intl.Locale.prototype.getNumberingSystems">
+      <h1>Intl.Locale.prototype.getNumberingSystems ( )</h1>
+      <p>When the `getNumberingSystems` method is called, the following steps are taken:</p>
+      <emu-alg>
+        1. Let _loc_ be the *this* value.
+        1. Perform ? RequireInternalSlot(_loc_, [[InitializedLocale]]).
+        1. Return NumberingSystemsOfLocale(_loc_).
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-Intl.Locale.prototype.getTimeZones">
+      <h1>Intl.Locale.prototype.getTimeZones ( )</h1>
+      <p>When the `getTimeZones` method is called, the following steps are taken:</p>
+      <emu-alg>
+        1. Let _loc_ be the *this* value.
+        1. Perform ? RequireInternalSlot(_loc_, [[InitializedLocale]]).
+        1. Let _region_ be GetLocaleRegion(_loc_.[[Locale]]).
+        1. If _region_ is *undefined*, return *undefined*.
```

Comment:
This logic seems like it belongs in TimeZonesOfLocale, replacing the assertion and bringing it in better alignment with LocaleDirection.
```patch
         1. Let _region_ be GetLocaleRegion(_loc_.[[Locale]]).
-        1. Assert: _region_ is not *undefined*.
+        1. If _region_ is *undefined*, return *undefined*.
         1. Let _list_ be a List of unique canonical time zone identifiers, which must be String values indicating a canonical Zone name of the IANA Time Zone Database, of those in common use in _region_. The list is empty if no time zones are commonly used in _region_. The list is sorted according to lexicographic code unit order.
         1. Return CreateArrayFromList( _list_ ).
```
```suggestion
```